### PR TITLE
Update sriov-network-operator crd and images

### PR DESCRIPTION
**Pull request by Mmduh-483**
Update sriov-network-operator crd to support switchdev and update operator and daemon images to be from harbor.mellanox

### CODE REVIEW

File: deployment/network-operator/charts/sriov-network-operator/values.yaml [hunk 1]
```diff
@@ -42,8 +42,8 @@ resources: {}
 
 # Image URIs for sriov-network-operator components
 images:
-  operator: quay.io/openshift/origin-sriov-network-operator:4.8
-  sriovConfigDaemon: quay.io/openshift/origin-sriov-network-config-daemon:4.8
+  operator: harbor.mellanox.com/cloud-orchestration/sriov-operator:latest
```
Comment (e0ne) [inline deployment/network-operator/charts/sriov-network-operator/values.yaml:45]: harbor.mellanox.com is available only inside Mellanox network. Can we use public registry here?

**Review by e0ne — CHANGES_REQUESTED** [review]

File: deployment/network-operator/charts/sriov-network-operator/values.yaml (hunk 1 above)
Comment (Mmduh-483) [reply deployment/network-operator/charts/sriov-network-operator/values.yaml:45]: You have a point
@moshe010 should I change it to mellanox dockerhub

Comment (moshe010) [reply deployment/network-operator/charts/sriov-network-operator/values.yaml:45]: yes let change to dockerhub

Comment (moshe010) [reply deployment/network-operator/charts/sriov-network-operator/values.yaml:45]: also can you tag the images and not use latest

**Review by moshe010 — CHANGES_REQUESTED** [review]

File: deployment/network-operator/charts/sriov-network-operator/values.yaml (hunk 1 above)
Comment (Mmduh-483) [reply deployment/network-operator/charts/sriov-network-operator/values.yaml]: updated

**Review by moshe010 — APPROVED** [review]

**Pull request merged**